mod/search.php

```diff
@@ -214,7 +214,7 @@ function search_content(App $a) {
 			FROM `item` %s
 			WHERE %s AND (`item`.`uid` = 0 OR (`item`.`uid` = %s AND NOT `item`.`global`))
 				$sql_extra
-			GROUP BY `item`.`uri` ORDER BY `item`.`id` DESC LIMIT %d , %d",
+			GROUP BY `item`.`uri`, `item`.`id` ORDER BY `item`.`id` DESC LIMIT %d , %d",
```

Comment:
In my opinion, it would be good if one of you could test this pull request on your system and then merge. Going through all requests will likely be a slightly longer process.